route_test.go

```diff
@@ -1339,3 +1340,67 @@ func TestMTURouteAddDel(t *testing.T) {
 		t.Fatal("Route not removed properly")
 	}
 }
+
+func TestRouteViaAddDel(t *testing.T) {
+	_, err := RouteList(nil, FAMILY_V4)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	tearDown := setUpNetlinkTest(t)
+	defer tearDown()
+
+	link, err := LinkByName("lo")
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if err := LinkSetUp(link); err != nil {
+		t.Fatal(err)
+	}
+
+	route := &Route{
+		LinkIndex: link.Attrs().Index,
+		Dst: &net.IPNet{
+			IP:   net.IPv4(192, 168, 0, 0),
+			Mask: net.CIDRMask(24, 32),
+		},
+		MultiPath: []*NexthopInfo{
+			{
+				LinkIndex: link.Attrs().Index,
+				Via: &Via{
+					AddrFamily: FAMILY_V6,
+					Addr:       net.ParseIP("2001::1"),
+				},
+			},
+		},
+	}
+
+	if err := RouteAdd(route); err != nil {
+		t.Fatalf("route: %v, err: %v", route, err)
+	}
+
+	routes, err := RouteList(link, FAMILY_V4)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if len(routes) != 1 {
+		t.Fatal("Route not added properly")
+	}
+
+	if !cmp.Equal(route.MultiPath[0].Via, routes[0].Via) {
```

Comment:
do we really need to import this go-cmp/cmp to compare a two fields structure also considering the structure already has a `Equal()` method? 

Unless widely used and needed, I'd prefer to keep this project lean from outside imports

Reply:
Nope, it can be removed.